src/Classes/ServiceAPI/MyRadio_Season.php

```diff
@@ -257,22 +257,30 @@ public static function create($params = [])
 
         //Same with tags
         if (!empty($params['tags'])) {
-            $tags = explode(' ', $params['tags']);
+            // Explode the tags
+            // We want to handle the case when people delimit with commas, spaces, or commas and
+            // spaces, as well as handling extended spaces.
+            $tags = preg_split('/[, ] */', $params['tags'], null, PREG_SPLIT_NO_EMPTY);
             foreach ($tags as $tag) {
                 if (empty($tag)) {
                     continue;
+                } else if (strlen($tag) < 25) {
```

Comment:
invert this condition so the error handling is together (and the diff is smaller)